util/radec.py

```diff
@@ -0,0 +1,72 @@
+# Utilities for dealing with ra and dec
```

Comment:
This already exists in `pipeline/utils.py`. 
Clearly we have too many util files, and should consider unifying them. 
Perhaps this would be better done in a separate PR. 
For now, you can leave this if you feel these functions are better, and we can decide later which file to merge into the other.